### PR TITLE
configure sentry for ruby

### PR DESCRIPTION
**Pull request by samandmoore**
this sets up the new sentry ruby and rails stuff. it's nice.

### CODE REVIEW

File: app/controllers/rooms_controller.rb [hunk 1]
```diff
@@ -1,16 +1,17 @@
 class RoomsController < ApplicationController
-    def index
-        @room = Room.new
-        render :index, layout: nil
-    end
+  def index
+    @room = Room.new
+    raise 'oops'
```
Comment (samandmoore) [inline app/controllers/rooms_controller.rb:4]:
```suggestion
```

File: config/initializers/sentry.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Sentry.init do |config|
+  config.breadcrumbs_logger = [:sentry_logger]
```
Comment (samandmoore) [inline config/initializers/sentry.rb:2]: this gets us nice stuff

image

Comment (kfinn) [reply config/initializers/sentry.rb:2]: love it

**Review by kfinn — COMMENTED** [review]
perfect. thank you!

**Review by kfinn — APPROVED** [review]

**Pull request merged**